src/components/LandingPage.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../assets/css/style.css';
+import logo from '../assets/images/logo.png';
+
+const landingPage = () => (
+  <div className="row">
+    <div className="column-left" style={{ backgroundColor: 'white' }}>
+      <div className="main-logo">
```

Comment:
@nziokaivy 
Is it a good idea to have inline styles?
Maybe we can place this style: `backgroundColor: 'white'` in external styles